lib/matplotlib/mathtext.py

```diff
@@ -2504,7 +2504,8 @@ def symbol(self, s, loc, toks):
                     break
             # Binary operators at start of string should not be spaced
             if (c in self._binary_operators and
-                    (len(s[:loc].split()) == 0 or prev_char == '{')):
+                    (len(s[:loc].split()) == 0 or prev_char == '{' or
```

Comment:
Yes, `_left_delim` is definitely more general than `(`. However, `{` is not in `_left_delim` (only `\{`), so I'll do that comparison separately.